wfe2/wfe.go

```diff
@@ -1373,12 +1387,46 @@ func (wfe *WebFrontEndImpl) addIssuingCertificateURLs(response http.ResponseWrit
 type orderJSON struct {
 	Status         core.AcmeStatus
 	Expires        time.Time
-	CSR            core.JSONBuffer
+	Identifiers    []core.AcmeIdentifier
 	Authorizations []string
+	FinalizeURL    string
 	Certificate    string `json:",omitempty"`
 	Error          string `json:",omitempty"`
 }
 
+// orderToOrderJSON converts a *corepb.Order instance into an orderJSON struct
+// that is returned in HTTP API responses. It will convert the order names to
+// DNS type identifiers and additionally create absolute URLs for the finalize
+// URL and the ceritificate URL as appropriate.
+func (wfe *WebFrontEndImpl) orderToOrderJSON(request *http.Request, order *corepb.Order) orderJSON {
+	// Convert the corepb.Order.Names back into identifiers for display to the
+	// user. We do this rather than send back newOrderRequest.Identifiers because
```

Comment:
I think this sentence may be left over from a previous iteration? There's no `newOrderRequest` in this scope, and we use this utility function in both NewOrder and getOrder.

Reply:
Yup! You're right. This is some spilled copy pasta :spaghetti: